player/src/bin/play.rs

```diff
@@ -82,7 +82,7 @@ fn main() {
                 )
                 .expect("Unable to find an adapter for selected backend");
 
-            let info = gfx_select!(adapter => global.adapter_get_info(adapter));
+            let info = gfx_select!(adapter => global.adapter_get_info(adapter)).unwrap();
```

Comment:
Not in scope, but we likely want an `unwrap_pretty` type thing here too.

Reply:
These are actually not expected to fail at all, unless we are hand-writing them.
I would be happy to accept such PR anyway though :)